schemas/com.snowplowanalytics.oss/oss_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Telemetry schema",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.telemetry",
+    "name": "heartbeat",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "userProvidedId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 256,
+      "description": "Optional ID provided by user as an input to each component. Intended to tie events together across modules, infrastructure and apps when used consistently"
+    },
+    "autoGeneratedId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 256,
+      "description": "ID automatically generated upon running a modules deployment script. Intended to identify each independent module, and the infrastructure it controls"
+    },
+    "moduleName": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 256,
+      "description": "The name of the terraform module"
+    },
+    "moduleVersion": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 32,
+      "description": "The version of the terraform module e.g. 1.0.0"
+    },
+    "instanceId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 256,
+      "description": "A unique identifier for the VM instance. Unique for each instance of the app running within a module"
+    },
+    "appGeneratedId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "pattern": "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
+      "description": "Automatically generated UUID on every service restart"
+    },
+    "cloud": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 256,
+      "description": "The cloud AWS or GCP, none for on-premise"
```

Comment:
Could be `"enum": ["aws", "gcp", "azure", null]`?

If we go the `enum` way - we still should preserve nullable `string` type, because Iglu does not always infer the right type for enums.